onesignalclient/notification.py

```diff
@@ -50,26 +54,32 @@ def include_player_ids(self, value):
     def include_segments(self):
         return self._include_segments
 
-    # notification.include_segments = [Notification.SEGMENT_ALL]    
-    @include_segments.setter
-    def include_segments(self, value):
+    def _validate_include_segments(self, value):
         if self.mode != self.SEGMENTS_MODE:
             raise TypeError('Mode should be set to segment.')
 
         if not isinstance(value, list):
             raise TypeError('Value must be a list.')
         
```

Comment:
Can you remove these unwanted whitespaces?

Ref. 'W293:blank line contains whitespace'